Public/Src/Cache/ContentStore/DistributedTest/Sessions/RedisDistributedContentTests.cs

```diff
@@ -397,10 +397,12 @@ public Task RepeatedBlobIsSkipped()
                     var fileString = Encoding.Default.GetString(file);
 
                     await session0.PutContentAsync(context, fileString).ShouldBeSuccess();
+                    await Task.Delay(500); // Wait for put blob

```

Comment:
>await Task.Delay(500); // Wait for put blob [](start = 20, length = 43)

Rather than wait for a static set of time, could you query for success in a loop? This test will be run on machines which are maxed at 100%, and 500ms might not be enough time for it to complete the FireAndForget task.

Reply:
Maybe add a config (or event static variable) that tells it to PutBlob inline.

---
In reply to: [340240813](https://github.com/microsoft/BuildXL/pull/1101#discussion_r340240813) [](ancestors = 340240813)